build.gradle

```diff
@@ -192,17 +189,6 @@ subprojects { project ->
       apply plugin: "java-library"
       apply plugin: "kotlin"
 
-      /*
-       * Apply the Animal Sniffer plugin to check that code is Android compatible.
-       */
-
-      apply plugin: "ru.vyarus.animalsniffer"
```

Comment:
@io7m are you actively using this? It seemed out of date and unused so I opted to remove it. However, I'm happy to revert this if it's something you're actively using.

Reply:
Technically we're still using it for reasons of paranoia. Some of the modules are jar-typed rather than aar-typed, and this just adds an extra layer of safety to ensure we aren't calling APIs that aren't supported by Android.

It's not essential.

Reply:
No problem, I reverted this! Just wanted to see if there was any cruft I could clear out.